modules/sdk-coin-apt/test/unit/transactionBuilder/digitalAssetTransferBuilder.ts

```diff
@@ -7,138 +7,280 @@ import { DigitalAssetTransfer } from '../../../src/lib/transaction/digitalAssetT
 
 describe('Apt Digital Asset Transfer Builder', () => {
   const factory = getBuilderFactory('tapt');
-  //TODO: change coin 'tapt' to digital asset (nft) when configured
-  it('should build a digital asset transfer', async function () {
-    const digitalAssetTransfer = new DigitalAssetTransfer(coins.get('tapt'));
-    const txBuilder = factory.getDigitalAssetTransactionBuilder(digitalAssetTransfer);
-    txBuilder.sender(testData.sender2.address);
-    txBuilder.recipient(testData.digitalTokenRecipients[0]);
-    txBuilder.gasData({
-      maxGasAmount: 200000,
-      gasUnitPrice: 100,
+  describe('Backward Compatibility with Recipient', () => {
```

Comment:
please prevent code duplicity.
remove the two different set of tests i.e. `Backward Compatibility with Recipient` and `Forward Compatibility with Recipients`.
and extend the existing tests only.